### DISCUSSION TITLE
Launch apps using keyboard shortcuts

### DISCUSSION THREAD

**Discussion by gohma231** [lawand-dot-io/taskbar#43] [Feature requests]
In windows, you can pin an app to the task bar and drag it all the way to the left. If it is the leftmost app, you can launch using "WIN + 1". Apps next to that icon can be launched using "WIN + 2", WIN + 3", etc....

I use this feature all the time in windows and this would be really helpful if added!

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/197

**Comment by rishi-valon** [comment]
Just commenting that this would be absolutely incredible!
